roles/promtail/files/config.yml

```diff
@@ -0,0 +1,35 @@
+server:
+  http_listen_port: 3100
+  grpc_listen_port: 0
+
+positions:
+  filename: /data/promtail/positions.yaml
+
+clients:
+  - url: http://172.16.34.1:3100/loki/api/v1/push
+# hardcoded, gotta replace by var in ansible playbook and lineinfile
+ 
+scrape_configs:
+- job_name: journal
+  journal:
+    json: false
+    max_age: 12h
+    path: /var/log/journal
+    labels:
+      #<custom label>
+      job: systemd-journal
+  relabel_configs:
+    - source_labels: ['__journal__systemd_unit']
+      target_label: 'unit'
+    - source_labels: ['__journal_syslog_identifier']
+      target_label: 'syslog_identifier'
+    - source_labels: ['__journal__hostname']
+      target_label: 'hostname'
+    - source_labels: ['__journal_priority']
+      target_label: 'log_level'
+  pipeline_stages:
+  - template:
+      source: log_level
+      template: '{{ if eq .Value "0" }}{{ Replace .Value "0" "emerg" 1 }}{{ else if eq .Value "1" }}{{ Replace .Value "1" "alert" 1 }}{{ else if eq .Value "2" }}{{ Replace .Value "2" "crit" 1 }}{{ else if eq .Value "3" }}{{ Replace .Value "3" "err" 1 }}{{ else if eq .Value "4" }}{{ Replace .Value "4" "warning" 1 }}{{ else if eq .Value "5" }}{{ Replace .Value "5" "notice" 1 }}{{ else if eq .Value "6" }}{{ Replace .Value "6" "info" 1 }}{{ else if eq .Value "7" }}{{ Replace .Value "7" "debug" 1 }}{{ end }}'
```

Comment:
Wow!

Reply:
I assumed that this is better than just a number

Reply:
It is!